playground/test_func.py

```diff
@@ -0,0 +1,97 @@
+from onnxscript import script
+from onnxscript.onnx_opset import opset17 as op
+from onnxscript import onnx_types as ot
+
+import numpy as np
+from typing import Any, Dict, Iterator, List, Optional, AbstractSet, Tuple
+
+import torch
+from torch.testing._internal import common_methods_invocations
+from torch.testing._internal.opinfo.core import OpInfo
+
+TESTED_DTYPES = (
+    # torch.bool,
+    # torch.uint8,
+    # torch.int8,
+    # torch.int16,
+    # torch.int32,
+    # torch.int64,
+    # Floating types
+    # torch.float16,
+    torch.float32,
+    torch.float64,
+    # torch.bfloat16,
+    # Complex types
+    # torch.complex32,
+    # torch.complex64,
+    # torch.complex128,
+)
+
+# from pyinstrument import Profiler
+
+# profiler = Profiler()
+# profiler.start()
+
+@script()
+def LeakyRelu(input, negative_slope: float = 0.01, inplace: bool = False):
+    zero = op.CastLike(0, input)
+    negative_slope = op.CastLike(negative_slope, input)
+    return op.Where(input < zero, negative_slope * input, input)
+
+
+@script()
+def FuncWithStr(input, negative_slope: float = 0.01, mode: str = "foo"):
+    if mode == "foo":
+        return input
```

Comment:
FYI: 
(a) `return` inside control-flow constructs is not supported.
(b) identity assignment is not supported (though probably easy to add)
Suggest using
```py
   if mode == "foo":
      result = op.Identity(input)
   else:
      result = ...
   return result
```

Reply:
Perfect, thanks!